src/main/java/selenium/pages/LoginPageWithPF.java

```diff
@@ -0,0 +1,33 @@
+package selenium.pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import selenium.Driver;
+import selenium.TestUser;
+
+public class LoginPageWithPF extends BasePage {
+    @FindBy(id = "user-name")
+    private WebElement UsernameInput;
+    @FindBy(id = "password")
+    private WebElement PasswordInput;
+    @FindBy (id = "login-button")
+    private WebElement LoginButton;
```

Comment:
please add a blank lines between fields for better readability

Reply:
Done